### ISSUE TITLE
[Feature] OSD info display time

### ISSUE THREAD

**Issue by Liquidrag** [jurialmunkey/skin.arctic.horizon.2#501]
### Skin section

Fullscreen Video OSD

### Feature description

First of all, I love this skin and the progression you have made. 

Currently, I have autoclose when paused and my interval is set to 5 seconds, which works exactly as I want. I'm hoping that when I push down on "info" the OSD does not go away. 

### Why is this feature needed? Please justify and explain.

So that I do not have to hit the info button which pulls up the actual Info screen. I want to see the small one at the bottom you have. 

### Screenshots and Additional Info

What I want to see

<img width="1285" alt="Screen Shot 2022-10-02 at 8 28 20 AM" src="https://user-images.githubusercontent.com/43726430/193456949-3dbde5b6-5e57-40cf-b754-2b3863001642.png">

What I hope not to see but only when I hit the info button. 

<img width="1281" alt="Screen Shot 2022-10-02 at 8 31 33 AM" src="https://user-images.githubusercontent.com/43726430/193457040-515b28dc-a2bb-4ca7-9302-be77100a61a6.png">

### Checklist

- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
I think I actually ended up fixing this inadvertently with a previous change, but I've added in some extra safety checks to cancel the autoclose alarm just in case.

**Comment by Liquidrag** [comment]
> I think I actually ended up fixing this inadvertently with a previous change, but I've added in some extra safety checks to cancel the autoclose alarm just in case.

Yes it was fixed previously. I figured you just forgot to make it as complete. Thank you again!
